### ISSUE TITLE
conformance: Github action not building image with git version

### ISSUE THREAD

**Issue by jdolitsky** [opencontainers/distribution-spec#100]
For some reason, Github action does not have access to the .git directory: 
```
Step 5/9 : RUN make conformance-binary && mv output/conformance.test /conformance.test
 ---> Running in 66d264dd2646
fatal: not a git repository (or any of the parent directories): .git
cd conformance && \
	CGO_ENABLED=0 go test -c -o /go/src/github.com/opencontainers/distribution-spec/output//conformance.test \
		--ldflags="-X github.com/opencontainers/distribution-spec/conformance.Version="
go: downloading github.com/opencontainers/distribution-spec v1.0.0-rc0.0.20200108182153-219f20cbcfa1
```

Perhaps we use build args instead of relying on git?

**Comment by jdolitsky** [comment]
Once #104 is merged, build args will be available, but im not seeing ways to do this with github actions..

I think a more sensible approach would be to point to a remote image (via `docker://` syntax, see [here](https://help.github.com/en/actions/building-actions/metadata-syntax-for-github-actions#using)), that is pushed by this repo, tagged to some release. Maybe even move the action into its own repo "distribution-spec-conformance-action"

**Comment by jdolitsky** [comment]
Related GitHub support ticket: https://github.community/t5/GitHub-Actions/Feature-Request-Build-args-support-in-Docker-container-actions/m-p/37802

**Comment by vbatts** [comment]
@jdolitsky is this still an issue?

**Comment by jdolitsky** [comment]
@vbatts not quite. But waiting to use the github image until a release is made: https://github.com/opencontainers/distribution-spec/blob/master/action.yml#L11

Will rename this to reflect
